hash/x68k_flop.xml

```diff
@@ -16387,6 +16387,54 @@ Typing "CD PACMAN" and enter key, Typing "PAC" (or "PAC.BAT") and enter key. The
 		</part>
 	</software>
 
+	<software name="sxwin31b" cloneof="sxwin31">
+		<description>SX-Window v3.1 (Alt 2)</description>
```

Comment:
`alt `should be lowercase

Reply:
Done. Should I change the other alt to `SX-Window v3.1 (alt)` to keep it consistent?

Reply:
If there is one then yes, thanks.

Reply:
Done!